docs-v2/components/homepage/CoreValueCard.tsx

```diff
@@ -0,0 +1,29 @@
+import { Heading, Icon, majorScale, Pane, Paragraph, useTheme } from "evergreen-ui"
+
+interface Props {
+  icon: typeof Icon,
+  title: string,
+  description: string
+}
+
+const ValueCard = ({icon, title, description}: Props) => {
+  const { colors } = useTheme()
+
+  return (
+    <Pane marginX={majorScale(2)}>
+      <Icon
+        icon={icon}
+        size={majorScale(3)}
+        borderRadius="50%"
+        padding={majorScale(2)}
+        color={colors.blue500}
+        backgroundColor={colors.blue50}
+        marginBottom={majorScale(4)}
+      />
+      <Heading size={600} marginBottom={8}>{title}</Heading>
```

Comment:
Try to avoid using hard-coded here! Opt for `majorScale()` (8px) or `minorScale()` (4px). Makes it easy to find + change all at once when we cross the bridge for spacing tokens.